force-app/main/default/lwc/takeHomePayCalc/takeHomePayCalc.js

```diff
@@ -0,0 +1,557 @@
+  };
+
+   calculateGrossSalaryDivided = () => {
+    this.grossSalDiv = (this.salAmt/this.timesPerYear);
+    this.grossSalDiv = this.grossSalDiv.toFixed(2);
+    console.log("**Gross salary amount divided: " + this.grossSalDiv);
+  };
+
+
+  convertPayFrequencyToNumberPerYear() {
+    switch (this.payFrequencyVal) {
+      case "Weekly":
+        this.timesPerYear = 52;
+        break;
+      case "Biweekly":
+        this.timesPerYear = 26;
+        break;
+      case "SemiM":
+        this.timesPerYear = 24;
+        break;
+      case "Monthly":
+        this.timesPerYear = 12;
+        break;
+      case "Quarterly":
+        this.timesPerYear = 4;
+        break;
+      case "SemiY":
+        this.timesPerYear = 2;
+        break;
+      case "Yearly":
+        this.timesPerYear = 1;
+        break;
+      default:
+        console.log("**Error converting pay frequency to number");
+    }
+    console.log("**timesPerYear: " + this.timesPerYear);
+  }
+
+
+  calculateYearlyPretaxDeductions() {
+    const yrlyPretaxDeduct = this.addtlPretaxDeductAmt * this.timesPerYear;
+    console.log("**yrlyPretaxDeductionAmt: " + yrlyPretaxDeduct.toFixed(2));
+    return yrlyPretaxDeduct.toFixed(2);
+  }
+
+  calculateYearlyPosttaxDeductions() {
+    const yrlyPosttaxDeduct = this.addtlPosttaxDeductAmt * this.timesPerYear;
+    console.log("**yrlyPosttaxDeductionAmt: " + yrlyPosttaxDeduct);
+    return yrlyPosttaxDeduct.toFixed(2);
+  }
+
+  calculateYearlyTaxableIncome() {
+    if (this.salAmt === null && this.howPaidVal === "Hourly") {
+      this.calculateGrossYrlyHourlySalary();
+    }
+    console.log('**salAmt from calculateYrlyTaxableIncome: ' + this.salAmt);
+    const deduct = parseFloat(this.handleItemizedDeduction());
+    const preTax = parseFloat(this.calculateYearlyPretaxDeductions());
+    this.yrlyTaxableInc = Math.round(this.salAmt + this.addtlIncome - (deduct + preTax));
+  }
+
+  calculateExtraFedTaxPaid() {
+    const extraTaxPaid = Math.round(
+      this.extraTaxPerPayPeriod * this.timesPerYear
+    );
+    console.log("**Extra tax paid: " + extraTaxPaid);
+    return extraTaxPaid;
+  }
+
+  calculateYrlySocSecTax() {
+    if (this.socExemptValue === "Y") {
+      return 0.0;
+    } else {
+      let socSecTax = 0.0; 
+      if (this.yrlyTaxableInc < this.socSecMaxVal) {
+        socSecTax = (this.socSecRate / 100) * this.yrlyTaxableInc;
+      } else {
+        socSecTax = (this.socSecRate / 100) * this.socSecMaxVal;
+      }
+      return Math.round(socSecTax);
+    }
+  }
+
+  calculateYrlyMedicareTax() {
+    if (this.medExemptValue === "Y") {
+      return 0.0;
+    } else {
+      let mdcrTax = (this.medicareRate / 100) * this.yrlyTaxableInc;
+      return Math.round(mdcrTax);
+    }
+  }
+
+  async calculateYrlyFederalTax() {
+    let federalTax = 0.0;
+    if (this.fedExemptValue === "Y") {
+      return federalTax;
+    } else {
+      let taxBrName = this.maxTaxBracket;
+      if (taxBrName !== null && taxBrName !== "") {
```

Comment:
I think again that revising the tax bracket naming convention could make the logic built to handle the various calculations easier, but the logic in this method is sound and you did a great job constructing it!